### PR TITLE
copr: bit shift use wrapping shift instead checked shift

### PR DESCRIPTION
**Pull request by gauss1314**
Signed-off-by: gauss <gauss1314@gmail.com>

Fix the issue #5905 

###  What have you changed?

bit shift use wrapping shift instead checked shift

###  What is the type of the changes?

Pick one of the following and delete the others:

- Bugfix (a change which fixes an issue)

###  How is the PR tested?

- Unit test
- Integration test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No

###  Does this PR affect `tidb-ansible`?

No

### CODE REVIEW

File: components/tidb_query/src/expr/builtin_op.rs [hunk 1]
```diff
@@ -180,14 +180,22 @@ impl ScalarFunc {
...
     }
 
     pub fn right_shift(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<i64>> {
         let lhs = try_opt!(self.children[0].eval_int(ctx, row));
         let rhs = try_opt!(self.children[1].eval_int(ctx, row));
-        let ret = (lhs as u64).checked_shr(rhs as u32).unwrap_or(0);
+        let ret = if rhs as u64 >= 64 {
+            0
+        } else {
+            (lhs as u64).wrapping_shl(rhs as u32)
```
Comment (lonng) [inline components/tidb_query/src/expr/builtin_op.rs]:
```suggestion
            (lhs as u64).wrapping_shr(rhs as u32)
```

**Comment by gauss1314** [comment]
@sre-bot /run-integration-copr-test

**Comment by gauss1314** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/31

**Pull request merged**